SPECS/mock-core-configs/mock-core-configs.spec

```diff
@@ -1,54 +1,76 @@
-Summary:    Mock core config files basic chroots
+%if 0%{?el8}
+%global python3 /usr/libexec/platform-python
+%endif
+
 Name:       mock-core-configs
-Version:    36.4
+Version:    41.2
 Release:    1%{?dist}
-License:    GPLv2+
+Vendor:     Microsoft Corporation
+Distribution: Azure Linux
+Summary:    Mock core config files basic chroots
+
+License:    GPL-2.0-or-later
 URL:        https://github.com/rpm-software-management/mock/
 # Source is created by
 # git clone https://github.com/rpm-software-management/mock.git
 # cd mock/mock-core-configs
 # git reset --hard %%{name}-%%{version}
 # tito build --tgz
 Source:     https://github.com/rpm-software-management/mock/archive/refs/tags/%{name}-%{version}-1/%{name}-%{version}-1.tar.gz#/%{name}-%{version}.tar.gz
+BuildArch:  noarch
+
+# The mock.rpm requires this.  Other packages may provide this if they tend to
+# replace the mock-core-configs.rpm functionality.
+Provides: mock-configs
```

Comment:
Should this `Provide` be versionless?

Reply:
This was taken directly from the Fedora spec. I can add `= %{version}-%{release}` if that's preferred.

Reply:
I'm never fully sure about `Provides`, so I'm not going to pretend int's preferred. I low-key hoped you may simply know more here.:)